src/applications/vaos/containers/VAFacilityPage.jsx

```diff
@@ -41,14 +42,43 @@ const uiSchema = {
     'ui:title':
       'Appointments are available at the following locations. Some types of care are only available at one location. Select your preferred location',
     'ui:widget': 'radio',
+    'ui:validations': [
```

Comment:
I don't think this will ever be displayed, but figured it was a good thing to have in place.